modules/administration-guide/partials/con_che-operator.adoc

```diff
@@ -17,7 +17,10 @@ Creates and controls the necessary {orch-name} objects to run a {prod-short} ins
 `CheCluster` custom resource (CR)::
 On a cluster with the {prod-short} operator, it is possible to create a `CheCluster` custom resource (CR). The {prod-short} operator ensure full lifecycle management of the {prod-short} server components on this {prod-short} instance.
 
+The {prod-short} operator is also provides routing for {devworkspace} by configuring the `che-gateway`.
```

Comment:
```suggestion
```

Reply:
> che-operator is now part of devworkspace startup process where it provides routing for the devworkspace. It does so by configuring the che-gateway.

But devworkspace is installed only when you create a CheCluster CR, right?

Reply:
Ah, I mean workspace. It is just with devworkspace operator, you create workspace by creating Devworkspace CR so I've called it devworkspace. TBH I don't know what's the correct wording anymore. But here I meant really the user's workspaces, that he/she will create from the devfile.